factcast-spring-boot-autoconfigure/src/main/java/org/factcast/spring/boot/autoconfigure/core/FactCastSnapshotCacheAutoConfiguration.java

```diff
@@ -35,7 +34,7 @@ public class FactCastSnapshotCacheAutoConfiguration {
 
   @Bean
   @ConditionalOnMissingBean
-  public SnapshotCache snapshotCache(FactStore store) {
-    return new FactCastSnapshotCache(store);
+  public SnapshotCache snapshotCache() {
```

Comment:
Why is the noop one needed?

Reply:
Didn't we agree in having this as the default? so if someone tries to save a Snapshot using the previous default (factcast server) then it fails and they have to manually choose a implementation that suits their needs?